Dockerfile.stable

```diff
@@ -0,0 +1,18 @@
+FROM node:18-alpine AS base
+
+# Install pnpm through corepack
+RUN apk update
+RUN apk add --no-cache libc6-compat
+RUN corepack enable
+RUN corepack prepare pnpm@latest --activate
+
+# Specify a path for pnpm global root
+ENV PNPM_HOME=/usr/local/bin
+
+FROM base as runtime
+
+# Install stable version of momoka
+RUN pnpm add -g @lens-protocol/momoka
+
+# Run using the default shell, this is important to infer the $NODE_URL env variable
+CMD ["sh", "-c", "momoka --node $NODE_URL --environment=POLYGON --concurrency=100 --fromHead=true"]
```

Comment:
lets change this to

CMD ["sh", "-c", "momoka --node $NODE_URL --environment=$ENV --concurrency=$CONCURRENCY]

Reply:
That's fair - will do